### ISSUE TITLE
Support for tmpfs

### ISSUE THREAD

**Issue by markmsmith** [testcontainers/testcontainers-node#26]
It would be helpful to be able to specify a `tmpfs` setting, the same way you can with testcontainers-java:  
https://github.com/testcontainers/testcontainers-java/issues/673

My use case is that I'd like to point a test database's data volume at a tmpfs in order to make our unit tests run quicker.

Thanks.

**Comment by cristianrgreco** [comment]
Hi @markmsmith 

I'm planning to implement this similarly to testcontainers-java:

```typescript
const container = await new GenericContainer("...")
      .withTmpFs({ path: "rw" })
      .start();
```

**Comment by cristianrgreco** [comment]
Available in 1.3.0.

**Comment by markmsmith** [comment]
Confirmed working in 1.3.1 with the command
```ts
.withTmpFs({ '/temp_pgdata': 'rw,noexec,nosuid,size=65536k' })
```
